CHANGELOG.md

```diff
@@ -17,6 +17,61 @@ _If you are upgrading: please see [`UPGRADING.md`](UPGRADING.md#unreleased)._
 
 ### Fixed
 
+## [2.0.0] - 2025-06-xx
+
+### Added
+
+- ✨ Improved device and gateset intermediate representation to Qiskit's Target ([#560]) ([**@burgholzer**, **@nquetschlich**])
+- ✨ Improved Output Logic and supported Formats (e.g., QASM3) ([#518]) ([**@simon1hofmann**, **@burgholzer**, **@nquetschlich**])
+- ✨ Add arithmetic benchmarks ([#586]) ([**@simon1hofmann**, **@burgholzer**])
+- ✨ Add mirror circuit option for all benchmarks ([#577]) ([**@CreativeBinBag**, **@burgholzer**, **@nquetschlich**])
+- ✨ Add symbolic parameters for variation benchmarks ([#581]) ([**@nquetschlich**, **@burgholzer**])
+- ✨ Add distinct `get_benchmark` function per level ([#571]) ([**@simon1hofmann**, **@burgholzer**])
+- ✨ Add clifford+T gateset support ([#555]) ([**@simon1hofmann**, **@burgholzer**, **@nquetschlich**])
+- ✨ Add HHL algorithm ([#582]) ([**@nquetschlich**, **@burgholzer**])
+- ✨ Add Bernstein-Vazirani algorithm ([#505]) ([**@simon1hofmann**, **@burgholzer**, **@nquetschlich**])
+- ✨ Add two benchmarks from BMW's QUARK framework ([#541]) ([**@fkiwit**])
+
+### Changed
+
+- ✨ Add transpile call at target-independent level ([#580]) ([**@simon1hofmann**, **@burgholzer**])
+- ✨ Add registry for benchmarks, devices, and native gatesets ([#585, #572]) ([**@simon1hofmann**, **@burgholzer**])
+- ✨ Add Hatchling and dependency groups ([#530]) ([**@nquetschlich**, **@burgholzer**])
+- ✨ Re-add Python 3.9 support ([#531]) ([**@simon1hofmann**])
+- ✨ Add test support for Python 3.13 ([#533]) ([**@simon1hofmann**])
+- 🎨 Simplify imports ([#587]) ([**@simon1hofmann**])
+- 🎨 Update IonQ and Rigetti devices ([#570]) ([**@nquetschlich**, **@burgholzer**])
+- 🎨 Refactor Shor's benchmark ([#548]) ([**@simon1hofmann**])
```

Comment:
A little more context on _what_ has been refactored could be helpful here.

Reply:
Done.